### PR TITLE
Turn beluga_ros into a Python package

### PR DESCRIPTION
**Pull request by hidmic**
### Proposed changes

Precisely what the title says. A small addition to #332.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [x] 🚀 Feature (change which adds functionality)
- [ ] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [x] Lint and unit tests (if any) pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

File: beluga_ros/cmake/ament.cmake [hunk 1]
```diff
@@ -22,8 +22,14 @@ find_package(tf2 REQUIRED)
 find_package(tf2_eigen REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
 
-ament_python_install_package(${PROJECT_NAME} PACKAGE_DIR
-                             ${PROJECT_SOURCE_DIR}/tools)
+ament_python_install_package(
+  ${PROJECT_NAME}
+  PACKAGE_DIR
```
Comment (serraramiro1) [inline beluga_ros/cmake/ament.cmake:27]: Not sure how you came up with this :100: 

You rock!

**Comment by serraramiro1** [comment]
Can we address the merge conflict so we can merge this?

**Review by serraramiro1 — APPROVED** [review]

**Comment by hidmic** [comment]
Ready to go @serraramiro1.

File: beluga_ros/cmake/ament.cmake (hunk 1 above)
Comment (hidmic) [reply beluga_ros/cmake/ament.cmake:27]: When working with ROS 2, always look at the [sources](https://github.com/ament/ament_cmake/blob/rolling/ament_cmake_python/cmake/ament_python_install_package.cmake).

**Pull request merged**